runtime/lua/vim/lsp/protocol.lua

```diff
@@ -764,6 +764,14 @@ function protocol.make_client_capabilities()
             return res
           end)(),
         },
+        completionList = {
+          itemDefaults = {
```

Comment:
Note that this is indirectly breaking one of the still exposed lsp.utils methods that was recently deprecated. Part of the reason why I suggested to add itemDefaults later and why I wanted to fast track the removal of those methods.

I'm fine pushing ahead with this anyway, but if we do we could at least mention it in the news.txt

Reply:
`extract_completion_items` if called without the default application will likely no longer return usable results.

Reply:
I see. Okay yeah I'm not too worried given the util will be removed soon. I will document that in both the function and in `news.txt`.